### PR TITLE
Allow users not to submit lock file with the request

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thamos/cli.py [hunk 1]
```diff
@@ -344,6 +344,13 @@ def _print_version(ctx, json_output: bool = False):
     is_flag=True,
     help="Do not wait for analysis to finish, just submit it.",
 )
+@click.option(
+    "--no-user-stack",
+    is_flag=True,
```
Comment (saisankargochhayat) [inline thamos/cli.py:349]: r(the/this)?

Comment (fridex) [reply thamos/cli.py:349]: Adjusted in de5a705. Thanks! :+1:

**Review by saisankargochhayat — APPROVED** [review]
Thanks :100:

**Pull request merged**